src/materials/NEMLStressInput.C

```diff
@@ -0,0 +1,48 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/*                       BlackBear                              */
+/*                                                              */
+/*           (c) 2017 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#ifdef NEML_ENABLED
+#include "NEMLStressInput.h"
+
+registerMooseObject("BlackBearApp", NEMLStressInput);
+
+InputParameters
+NEMLStressInput::validParams()
+{
+  InputParameters params = NEMLStressBase::validParams();
+  params.addClassDescription("Use a constitutive model in the NEML library to compute stress");
```

Comment:
Make this string describe what's unique about this class. These strings shouldn't be the same for multiple classes (because they have to be doing something different from each other, right?)

Reply:
Now there is a single nemlstress class and it uses this string.